File: marcos006/programacao_basica/questao11.py
print("** Media ponderada dos alunos IFRN **")
nota1 = float(input("Digite a nota do primeiro bimestre: "))
nota2 = float(input("Digite a nota do segundo bimestre: "))
nota3 = float(input("Digite a nota do terceiro bimestre: "))
nota4 = float(input("Digite a nota do quarto bimestre: "))
nota_total = nota1 + nota2 + nota3 + nota4
media_ponderada = nota1 * 7.0 + nota2 * 5.0 + nota3 * 6.0 + nota4 * 9.5/nota_total
print("A média ponderada é de " + str(media_ponderada))

File: marcos006/programacao_basica/Questao11.java
import java.util.Scanner;

    public class Questao11 {
        public static void main(String[] args) {
            Scanner s;
            s = new Scanner(System.in);

        System.out.println("** Média ponderada dos alunos do IFRN **");
        
        System.out.println("Digite a nota do primeiro bimestre: ");
        float n1 = s.nextFloat();

        System.out.println("Digite a nota do segundo bimestre: ");
        float n2 = s.nextFloat();
    
        System.out.println("Digite a nota do terceiro bimestre: ");
        float n3 = s.nextFloat();

        System.out.println("Digite a nota do quarto bimestre: ");
        float n4 = s.nextFloat();

        float nt = n1 + n2 + n3 + n4;

        double mp = n1 * 7.0 + n2 * 5.0 + n3 * 6.0 + n4 * 9.5/nt;

        System.out.println("A média ponderada é de " + mp);

    }
}
